extensions/org-roam-dailies.el

```diff
@@ -139,55 +139,55 @@ In this case, interactive selection will be bypassed."
   (org-roam-dailies--capture (current-time) goto keys))
 
 ;;;###autoload
-(defun org-roam-dailies-goto-today ()
+(defun org-roam-dailies-goto-today (keys)
```

Comment:
I think this argument should be hidden as optional

Reply:
Fixed in 6c0df349b92cf272216f4c4b4d637b19ce602693